lib/ruby_lsp/executor.rb

```diff
@@ -523,7 +523,14 @@ def check_formatter_is_available
       # Syntax Tree will always be available via Ruby LSP so we don't need to check for it.
       return unless @store.formatter == "rubocop"
 
-      unless defined?(Support::RuboCopFormattingRunner)
+      rubocop_available = begin
+        require "rubocop"
```

Comment:
This is basically a copy of what we're doing in the formatting and diagnostic runner. Instead of repeating the require for rubocop, we can just move the require for the diagnostic and formatting runners to `internal`